README.md

```diff
@@ -21,6 +21,8 @@ To get a local copy up and running follow these simple example steps.
 
 - Ruby
 
+### project presentation [Video](https://drive.google.com/file/d/1_CRYO020mGg-_-eJVoyG2rWjquYgKl4X/view?usp=sharing)

```

Comment:
- [ ] Great work documenting your project! It would be great to have both authors here, and update the 'Install' and 'Using it locally' instructions.

Reply:
I am actually the only one on this i used the video we made last week course it the same project but we couldn't finish it last week